python/py-virtualenv/Portfile

```diff
@@ -22,38 +22,31 @@ homepage            https://virtualenv.pypa.io
 master_sites        pypi:v/${python.rootname}
 distname            ${python.rootname}-${version}
 
-checksums           rmd160  1ec5152c8d82350fd24358cfa0d112fdf70e7094 \
-                    sha256  0d62c70883c0342d59c11d0ddac0d954d0431321a41ab20851facf2b222598f3 \
-                    size    5121717
+checksums           rmd160  a3d2e5b06a9bfbc366571961966ccea70a74daca \
+                    sha256  7e4e234e1f27755685dc54063d989756790b0682d60e304db589fa1604938013 \
+                    size    7972183
 
 # keep older Python versions here, do add the EOL version to the list below
 python.versions     27 34 35 36 37 38
 
 if {${name} ne ${subport}} {
     # add EOL warnings
-    if {${python.version} in "34"} {
+    if {${python.version} in {"27" "34"}} {
```

Comment:
this can become ```if {${python.version} in "27 34"} {```, better readability in my opinion (what you might work as well, I haven't checked).